### DISCUSSION TITLE
How can we store the css styles into constant

### DISCUSSION THREAD

**Discussion by NarendraKuruva** [adobe/react-spectrum#6593] [General]
I've updated few styles and wrote it in `.css` file.

Now I would like to store them in a variable and use it like how it is being used in **Combobox** component like below

```
// Import like this
import textfieldStyles from "@adobe/spectrum-css-temp/components/textfield/vars.css";

// Later in the component
<ProgressCircle
  aria-label={stringFormatter.format("loading")}
  size="S"
  isIndeterminate
  className={`${textfieldStyles} spectrum-Textfield-circleLoader ${styles} spectrum-InputGroup-input-circleLoader`}
/>
```

Do I need to take any extra steps to be able to use `.css` files in this way?

@devongovett @snowystinger 

Any help would be appreciated. Thanks!

**Comment by snowystinger** [comment]
I think we're now talking about this in https://github.com/adobe/react-spectrum/discussions/6603 ? I'm going to close this one as a result
